packages/components/src/navigation/README.md

```diff
@@ -30,14 +78,65 @@ Navigation supports the following props.
 
 ### `data`
 
-- Type: `array`
-- Required: Yes
+-   Type: `array`
+-   Required: Yes
 
 An array of config objects for each menu item.
 
-### `initial`
+Config objects can be represented
+
+#### `config.title`
+
+-   Type: `string`
+-   Required: Yes
+
+A menu item's title.
+
+#### `config.id`
+
+-   Type: `string|Number`
+-   Required: Yes
+
+A menu item's id.
+
+#### `config.parent`
+
+-   Type: `string|Number`
+-   Required: No
+
+Specify a menu item's parent id.
```

Comment:
Maybe note that `root` will get added as the menu item's parent if none is provided.